fuel-p2p/src/config.rs

```diff
@@ -102,7 +103,7 @@ impl P2PConfig {
             allow_private_addresses: true,
             enable_random_walk: true,
             connection_idle_timeout: Some(Duration::from_secs(120)),
-            topics: vec![],
+            topics: vec![NEW_TX_GOSSIP_TOPIC.into()],
```

Comment:
I think others should be included by default as well, this was an oversight from my side

Reply:
Updated